django/river/common/analyzer/analyzer.py

```diff
@@ -148,28 +146,27 @@ def analyze_attribute(self, attribute_mapping):
     def analyze_input_group(self, mapping_group, parent_attribute):
         input_group = InputGroup(id_=mapping_group["id"], attribute=parent_attribute)
         parent_attribute.add_input_group(input_group)
-        for input_ in mapping_group["inputs"]:
-            if input_["static_value"]:
-                input_group.add_static_input(input_["static_value"])
+        for static_input in mapping_group["static_inputs"]:
```

Comment:
it does not concern this file but before passing a mapping to an Analyzer, I think we should deserialize the mappings extracted from the current batch with the MappingSerializer and type hint everything. It would be less error prone than dealing with a dict thank to intellisense.

Reply:
Agreed, we can do that in another PR, though
I'll write an issue